src/constants/globalStyle.ts

```diff
@@ -14,7 +14,7 @@ const GlobalStyle = createGlobalStyle`
     font-weight: 400;
     font-size: 16px;
     font-family: ${monaSans.style.fontFamily};
-    background: #141517;
+    background: #F1F1F1 !important;
```

Comment:
it might be better to remove this `!important` :)

Reply:
This overrides mantine's background color, so we can't remove it.